Server updates (#1173)
minio/docs: source/reference/minio-mc/mc-tag.rst

.. end-mc-tag-desc

> An object can have up to 10 custom tags.

Reviewer: 🤔 could be nice to have the three mentions of `10 tags` use identical wording. Although I only even noticed because the diff makes them visible near each other. Feel free to ignore this comment.

> An object can have up to 10 custom tags.

Reviewer 2: ```suggestion
MinIO supports adding up to 10 custom tags to an object.
```
